docs/v4/nlp-manager.md

```diff
@@ -0,0 +1,366 @@
+# NLP Manager
+
+The `NlpManager` is able to manage several languages. It manages the named entities, and trains the NLP classifier for each language. Once trained, `NlpManager` is ready to process utterances. It will try to guess the language, if one isn't provided when processing an utterance. During processing, the `NlpManager` will:
+
+- Identify the language
+- Classify the utterance, using Machine Learning (ML)
+- Find named, and/or default entities in the utterance
+- Limit entities to those described with variables in the highest scoring intent, _if present_ (variable syntax for intents uses wildcard operators `%entity_name%`)
```

Comment:
explain here or in ner @ & % syntax

Reply:
Good question ... in fact we have
* "{{ ... }}" for "context placeholder replacement" when it comes to answer processing
* "@entityname" in ner (configurable btw) to include entities in utterances and parse them out
* "%entityname%" is when I remember not used by defsult anymore but was same as @entityname in v3? Could that be?

Am I right? Then we should change that above text to state @entityname?

Reply:
I think % is used only in v3 examples but we'll double check in the current code. If it's like that we can leave @EntityName